### ISSUE TITLE
struct.error occurred when trying to export session string

### ISSUE THREAD

**Issue by viper-adnan** [pyrogram/pyrogram#814]
## Checklist
- [X] I am sure the error is coming from Pyrogram's code and not elsewhere.
- [X] I have searched in the issue tracker for similar bug reports, including closed ones.
- [X] I ran `pip3 install -U https://github.com/pyrogram/pyrogram/archive/master.zip` and reproduced the issue using the latest development version.

## Description
I am trying to export session string with lastest pyrogram version but it's throwing error

``` python
from pyrogram import Client


APP_ID = int(input("Enter APP ID here: "))
API_HASH = input("Enter API HASH here: ")
with Client(
      ":memory:",
      api_id=APP_ID,
      api_hash=API_HASH
    ) as app:
    session_str = app.export_session_string()
print(session_str)
```

## Traceback

```
Traceback (most recent call last):
  File "main.py", line 11, in <module>
    session_str = app.export_session_string()
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/pyrogram/sync.py", line 51, in async_to_sync_wrap
    return loop.run_until_complete(coroutine)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/pyrogram/methods/utilities/export_session_string.py", line 44, in export_session_string
    return await self.storage.export_session_string()
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/pyrogram/storage/storage.py", line 75, in export_session_string
    struct.pack(
struct.error: 'I' format requires 0 <= number <= 4294967295
```

**Comment by rozari0** [comment]
I am facing the same problem.

**Comment by viper-adnan** [comment]
> The problem seems to be related to changed telegram IDs format to int64 from int32. I've modified SESSION_STRING_FORMAT = ">B?256sI?" to ">B?256sQ?" in pyrogram/storage/storage.py locally to make it works.
> 
> The hard thing is how both to keep compatibility with previous struct format and use the new one

Pyrogram sided

**Comment by viper-adnan** [comment]
> The problem seems to be related to changed telegram IDs format to int64 from int32. I've modified SESSION_STRING_FORMAT = ">B?256sI?" to ">B?256sQ?" in pyrogram/storage/storage.py locally to make it works.
> 
> The hard thing is how both to keep compatibility with previous struct format and use the new one

try both whichever works

**Comment by delivrance** [comment]
Fixed with #818.
